Cargo.toml

```diff
@@ -14,6 +14,7 @@ description = "A command-line utility for easily compressing and decompressing f
 atty = "0.2.14"
 bstr = { version = "1.6.2", default-features = false, features = ["std"] }
 bzip2 = "0.4.4"
+bzip3 = { git = "https://github.com/bczhc/bzip3-rs", commit = "fa98dfce62d2271d0d3c91fd3592a912314a0df9" }
```

Comment:
Any chance we could use

```
bzip3 = { version = "0.8.1", features =  ["bundled"] }
```

Maybe that would help in the CI woes

Reply:
good catch! that will probably work

though I would recommend adding `bzip3/bundled` as a default feature instead of making it a hard dependency

Reply:
Neat! I'm trying using this one.

@freijon I hope you don't mind, but I'm directly using your branch while trying to solve CI problems.